### PR TITLE
Enable --log-level=info for docker daemons for scalability jenkins jo…

### PR DESCRIPTION
**Pull request by dchen1107**
…b and e2e test. (take 2)

### CODE REVIEW

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml [hunk 1]
```diff
@@ -145,6 +145,8 @@
                 export REGISTER_MASTER="true"
                 # Reduce logs verbosity
                 export TEST_CLUSTER_LOG_LEVEL="--v=2"
+                # TODO: Remove when we figure out #22166 and other docker potential slowness
+                export DOCKER_TEST_LOG_LEVEL="--log-level=info"
```
Comment (mikedanese) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: This is unneccessary

File: cluster/saltbase/salt/docker/docker-defaults [hunk 2]
```diff
@@ -7,5 +7,9 @@
 {% if pillar.get('network_provider', '').lower() == 'kubenet' %}
   {% set bridge_opts = "" %}
 {% endif -%}
-DOCKER_OPTS="{{grains_opts}} {{e2e_opts}} {{bridge_opts}} --iptables=false --ip-masq=false --log-level=warn"
```
Comment (mikedanese) [inline cluster/saltbase/salt/docker/docker-defaults]: We are changing the default. Is that ok?

Comment (dchen1107) [inline cluster/saltbase/salt/docker/docker-defaults]: reset this back to default

**Comment by mikedanese** [comment]
LGTM, I think this should be fine for GKE. cc @jlowdermilk @roberthbailey

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml (hunk 1 above)
Comment (mikedanese) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: This is unneccessary (sorry, deleted the comment accidentally)

Comment (dchen1107) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: This is weird. On my local branch, it is totally gone. But remote branch doesn't have that change. Let me rebase and repush again.

**Comment by dchen1107** [comment]
Rebased and repushed. PTAL?

**Comment by roberthbailey** [comment]
lgtm.

**Pull request merged**